lib/DDG/Spice/UV.pm

```diff
@@ -0,0 +1,54 @@
+package DDG::Spice::UV;
+# Use the web service provided by EPA to display the UV Index for locations in the US.
+# The web service is described here: http://www.epa.gov/enviro/facts/services.html#uvindex
+
+use DDG::Spice;
+
+spice is_cached => 1;
+
+name "UV Index";
+source "EPA - United States Environmental Protection Agency";
+description "Display the UV Index for the user's location in the US";
+primary_example_queries "uv", "uv index";
+category "location_aware";
+topics "everyday", "travel", "geography";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/UV.pm";
+attribution github => ["https://github.com/Bjoern", "Bjoern Guenzel"],
+            twitter => ["https://twitter.com/fractality", "Fractality"],
+            web => ["http://blinker.net", "Bjoern's Blog"];
```

Comment:
We use the second element for the name of the person--can you change it to this? Thanks!

```
attribution github => ["https://github.com/Bjoern", "Bjoern Guenzel"],
 twitter => ["https://twitter.com/fractality", "Bjoern Guenzel"],
 web => ["http://blinker.net", "Bjoern Guenzel"];
```